.github/workflows/ui-tests.yml

```diff
@@ -141,7 +141,7 @@ jobs:
           path: tests/output
 
       - name: Create status check
-        if: ${{ always() }}
+        if: ${{ github.event_name != 'pull_request' && always() }}
```

Comment:
this line makes no sense, if we need to create a status check for everything except pull requests then let's leave only the first part, if we need to create a status check always then let's keep the old logic.